src/Microsoft.DotNet.Interactive.SqlServer.Tests/MsSqlConnectionTests.cs

```diff
@@ -56,11 +56,18 @@ public async Task It_can_connect_and_query_data()
             events.Should().NotContainErrors();
 
             events.Should()
-                  .ContainSingle<DisplayedValueProduced>()
-                  .Which
-                  .FormattedValues
-                  .Should()
-                  .ContainSingle(f => f.MimeType == HtmlFormatter.MimeType);
+                .Contain(e => e is DisplayedValueProduced && e.As<DisplayedValueProduced>()
+                .FormattedValues
+                .First()
+                .MimeType
+                .Equals(PlainTextFormatter.MimeType));
```

Comment:
```suggestion
                .Be(PlainTextFormatter.MimeType));
```

This is a big gotcha when using FluentAssertions. The `Equals` call here is `object.Equals` and if it returns false, no exception will be thrown and the test will pass.

Reply:
Oh wait, would it be fine if Equals returns false here since it is inside Contains()?

Reply:
Returning false doesn't provide any signal to the test runner that there was a problem. The `Be` method will cause an exception to be thrown, which the test runner can see.